docs/index.txt

```diff
@@ -21,6 +21,9 @@ Having trouble? We'd like to help!
 
 * Report bugs with Django in our `ticket tracker`_.
 
+A link to the proper channels for getting help is always available in the
+bottom right corner of your screen, above the language and version selectors.
```

Comment:
I don't think this message is that useful, here or in the tutorial admonitions. It's mostly just telling users to click the button the FAQ getting help page, which this section kind of summarizes already.

Reply:
Thanks for your review @adamchainz!

The idea behind the ticket as I understood it was to tell the users that they can always find help on their screen without needing to go back to another page or whatever. They’d just have to click on the « Get help » button.

So this message was about pointing to the button I added, to make the user aware of it.

Reply:
From the ticket it looks like I'm agreeing with @felixxm wrote  "I'm not sure about 2.1."

My concerns are:

* It's extra words to read before users get to do something. I think the tutorial can be overwheming already.
* It's telling users about something already on their screen
* "bottom right corner of your screen" won't apply in many contexts - RTL languages, if the design changes (especially if a proper mobile theme is developed), screen readers
* In this particular section, bullet point one directed users to the FAQ ; this just tells them of another link to the same place.

Overall I'm not that fussed about this though, if you think it's better we can go with it.

Reply:
Thank you for expressing your concerns 🙂 

> * It's extra words to read before users get to do something

I agree but I feel the sentence is very short and doesn't get in the way too much.

> * It's telling users about something already on their screen

Yes, that's the point of this ticket: to point users to things they already have on screen but don't necessarily see. The "Getting Help" section is also already in the sidebar, but you don't really see it when you're reading through the tutorials:
* it's not always on the screen
* it's in the bottom of the sidebar so it's easily forgotten

> * "bottom right corner of your screen" won't apply in many contexts

That's very true, I didn't think of it. Do you have any proposition for a better phrasing? Maybe something only relative to the version and language switchers? Maybe that won't work well for screen readers... Adding to your points: it also doesn't work well for any doc built and displayed outside of djangoproject.com (like the docs directly built by sphinx).

> * bullet point one directed users to the FAQ

The link I added points to one specific section of the FAQ, not the whole index.


Overall I'd like to keep the mention here and in the admonitions, but I do agree that we can come with a better phrasing that covers your observations.

Reply:
I agree with Adam, please remove this sentence.

Reply:
What about the answer on your comment in the ticket?

> The idea behind 2.1 is to make the admonition more memorable by providing a non-disruptive action that can be taken immediately without really cutting the workflow

Although I can see it creates other problems, I feel like telling the user that they can always find how to get help on their screen without having to come back to some point is better aligned with the spirit of the ticket.

But if you insist I'll remove it 🙂

Reply:
The first sentence in admonitions already contains link to the `Getting Help` page (`` :ref:`Getting Help section of the FAQ <faq-getting-help>` ``). IMO a redundant sentence can be misleading and is unnecessary.